packages/dds/tree/src/simple-tree/nodeKeys.ts

```diff
@@ -3,9 +3,16 @@
  * Licensed under the MIT License.
  */
 
-import { LocalNodeKey, NodeKeyIndex, NodeKeyManager, StableNodeKey } from "../node-key/index.js";
+import {
+	LocalNodeKey,
+	NodeKeyIndex,
+	NodeKeyManager,
+	StableNodeKey,
+	// eslint-disable-next-line import/no-internal-modules
```

Comment:
this code should not need to depend on internals of other libraries. Wither import these from somewhere thats allowed (feature-libraries)

Reply:
Updated illegal imports